File: PHP PowerPoint Examples/TableApiDemo/setSlideTableColumnWidth.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\TableApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new TableApi($configuration);

$name = "SetSlideTableColumnWidth.pptx"; 
$password = null; 
$folder = "Table"; 
$storage = null; 
$slideIndex = 0;
$shapeIndex = "0";
$columnIndex = 1;
$width = 10;
$apiInstance->setSlideTableColumnWidth($name, $slideIndex, $shapeIndex, $columnIndex, $width, $password, $folder, $storage);

File: Python PowerPoint Examples/TableApiDemo/setSlideTableColumnWidth.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.table_api.TableApi(configuration)

fileName = "SetSlideTableColumnWidth.pptx"
password = ""
storage = ""
slide_index = 0
shape_index = 0
column_index = 1
width = 10
inputFolder = "input"
result =api.set_slide_table_column_width(fileName, slide_index, shape_index, column_index=column_index, width=width,password=password, storage=storage, folder=inputFolder)